services/cinema_hall.py

```diff
@@ -0,0 +1,19 @@
+from typing import List
+
+from db.models import CinemaHall
+
+
+def get_cinema_halls() -> List[CinemaHall]:
```

Comment:
You do not return a `list`, you return a `QuerySet`. Please change your return type annotation to represent that here and in the other methods as well.